src/test/resources/grouper.test.properties

```diff
@@ -1,5 +1,5 @@
-groupings.api.test.uh-usernames=testiwta,testiwtb,testiwtc,testiwtd,testiwte
-groupings.api.test.uh-numbers=99997010,99997027,99997033,99997043,99997056
+groupings.api.test.uh-uids=testiwta,testiwtb,testiwtc,testiwtd,testiwte
```

Comment:
Make sure to update `groupings.api.test.uh-uids` to `groupings.api.test.uids` here too.